backend/coins/btc/account_test.go

```diff
@@ -95,4 +109,39 @@ func TestAccount(t *testing.T) {
 	require.Equal(t, accounts.OrderedTransactions{}, transactions)
 
 	require.Equal(t, []*btc.SpendableOutput{}, account.SpendableOutputs())
+
+	// check the number of available addresses for native and wrapped segwit.
+	require.Equal(t, len(account.GetUnusedReceiveAddresses()[0].Addresses), 20)
+	require.Equal(t, *account.GetUnusedReceiveAddresses()[0].ScriptType, signing.ScriptTypeP2WPKHP2SH)
+	require.Equal(t, len(account.GetUnusedReceiveAddresses()[1].Addresses), 20)
+	require.Equal(t, *account.GetUnusedReceiveAddresses()[1].ScriptType, signing.ScriptTypeP2WPKH)
+
+	// Create a new insured account.
+	account2 := btc.NewAccount(
```

Comment:
nit: seems like a separate unit test `TestInsuredAccount(...)` would be a bit cleaner/easier